.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,7 +24,7 @@
 2. 
 
 #### Steps to test this PR:
-1. Load up [this PR](https://mautibox.com)
+1. [Click here](https://mautibox.com), it will load the PR on Mautibox, an environnement automatically created to test PRs. No developer skill is needed.
```

Comment:
Maybe say 'click here and choose the PR from the dropdown to launch a test environment, automatically created ...'